src/main/java/dao/Columns.java

```diff
@@ -0,0 +1,32 @@
+package dao;
+
+import java.util.Map;
+
+public class Columns {
```

Comment:
why do we need this class, and why it is in `dao` package?
let's better remove it

Reply:
We need it to work with dao columns